Interfaces/API/test/Test_JobAPI.py

```diff
@@ -0,0 +1,60 @@
+""" Basic unit tests for the Job API
+"""
```

Comment:
Teh basic test was not successful, and actually was not activated. It does not look very useful to me. Anyway, I have updated the huge string to compare with and merged the two tests